templates/success.tmpl

```diff
@@ -0,0 +1,76 @@
+<html>
+<head>
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+    <link rel="stylesheet" href="https://s3.csh.rit.edu/csh-material-bootstrap/4.0.0-beta.3/dist/csh-material-bootstrap.min.css" media="screen">
+    <style>
```

Comment:
I think it would be nice if you could move this to a static CSS file. Maybe make a directory in the same level as `templates` called `static`?

Here's an example of how to serve static files with Gin: https://github.com/gin-gonic/gin#serving-static-files

Be careful with the paths, when I was writing out the Openshift container image for Golang+Gin I had to add some special environment variable to get the paths right. You can find that usage on main.go:30

Reply:
In line styles aren't a fun time, especially when its more than one quick thing.